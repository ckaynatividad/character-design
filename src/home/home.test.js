import { render, screen } from '@testing-library/react';
import Home from './home';
import Preview from '../components/preview/preview';
import Editor from '../components/editor/editor';

test('renders editor and preview in home panel', () => {
  const container = render(
    <><Editor /><Preview /></>
  );
  expect(container).toMatchSnapshot();
})