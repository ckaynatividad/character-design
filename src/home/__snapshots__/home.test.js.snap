// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders editor and preview in home panel 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="editor"
      >
        <div
          class="form-control"
        >
          <label>
            Head 
          </label>
          <select>
            <option
              value="bird"
            >
              bird
            </option>
            <option
              value="duck"
            >
              duck
            </option>
            <option
              value="dog"
            >
              dog
            </option>
            <option
              value="horse"
            >
              horse
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <label>
            Body 
          </label>
          <select>
            <option
              value="blue"
            >
              blue
            </option>
            <option
              value="fancy"
            >
              fancy
            </option>
            <option
              value="pink"
            >
              pink
            </option>
            <option
              value="red"
            >
              red
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <label>
            Bottom 
          </label>
          <select>
            <option
              value="single-leg"
            >
              single leg
            </option>
            <option
              value="blue-jeans"
            >
              blue jeans
            </option>
            <option
              value="pant-leg"
            >
              pant leg
            </option>
          </select>
        </div>
        <div
          class="form-control"
        >
          <label
            for="phrase"
          >
            Add a catch phrase 
          </label>
          <input
            name="phrase"
            type="text"
            value=""
          />
          <button>
            Add
          </button>
        </div>
      </div>
      <div
        class="preview"
      >
        <img
          src="/assets/undefined-head.png"
        />
        <img
          src="/assets/undefined-middle.png"
        />
        <img
          src="/assets/undefined-pants.png"
        />
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="editor"
    >
      <div
        class="form-control"
      >
        <label>
          Head 
        </label>
        <select>
          <option
            value="bird"
          >
            bird
          </option>
          <option
            value="duck"
          >
            duck
          </option>
          <option
            value="dog"
          >
            dog
          </option>
          <option
            value="horse"
          >
            horse
          </option>
        </select>
      </div>
      <div
        class="form-control"
      >
        <label>
          Body 
        </label>
        <select>
          <option
            value="blue"
          >
            blue
          </option>
          <option
            value="fancy"
          >
            fancy
          </option>
          <option
            value="pink"
          >
            pink
          </option>
          <option
            value="red"
          >
            red
          </option>
        </select>
      </div>
      <div
        class="form-control"
      >
        <label>
          Bottom 
        </label>
        <select>
          <option
            value="single-leg"
          >
            single leg
          </option>
          <option
            value="blue-jeans"
          >
            blue jeans
          </option>
          <option
            value="pant-leg"
          >
            pant leg
          </option>
        </select>
      </div>
      <div
        class="form-control"
      >
        <label
          for="phrase"
        >
          Add a catch phrase 
        </label>
        <input
          name="phrase"
          type="text"
          value=""
        />
        <button>
          Add
        </button>
      </div>
    </div>
    <div
      class="preview"
    >
      <img
        src="/assets/undefined-head.png"
      />
      <img
        src="/assets/undefined-middle.png"
      />
      <img
        src="/assets/undefined-pants.png"
      />
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
